code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -216,13 +228,9 @@
 /obj/item/melee/baton/wash(mob/living/user, atom/source)
 	if(turned_on && cell?.charge)
 		flick("baton_active", source)
-		user.Stun(stunforce)
-		user.Weaken(stunforce)
-		user.SetStuttering(stunforce)
-		deductcharge(hitcost)
+		baton_stun(user, user, TRUE)
```

Comment:
```suggestion
		baton_stun(user, user, skip_cooldown = TRUE)
```
to be consistent with how it's called with clumsiness.

Separately, should this check `baton_stun` return value? We can fail to stun ourselves because we are already stunned (due to the trait thingy), so i wonder if the extra "you are a dumdum and shock yourself" would be confusing?..